### ISSUE TITLE
Doesn't work with Hover Text (Apple accessibility feature)

### ISSUE THREAD

**Issue by OneSunnyDay** [tomasf/caffeine#3]
Hover Text is an accessibility feature, first introduced in Catalina, which allows the user to see information about the item their cursor is hovering over in very large text, as an aid to the visually impaired. 

When Hover Text is turned on (System Preferences —> Accessibility —> Zoom —> Enable Hover Text) and activated, and the user hovers their cursor over a menu item, they will typically see displayed in large text the name of that menu item and its current status. This can be very helpful for the visually impaired, who may not be able to see/read small icons and text, but who still needs to use a computer.

When hovering over Caffeine 1.1.3, Hover Text simply says “Menu Extra.” It would be very helpful if, instead, it is said “Caffeine” and stated which settings were active, if any.
